protocol/vitest.config.ts

```diff
@@ -4,10 +4,8 @@ import tsconfigPaths from 'vite-tsconfig-paths'
 export default defineConfig({
   plugins: [tsconfigPaths()],
   test: {
-    testTimeout: 30000, // 30 seconds
```

Comment:
Defining this globally for all tests is not great, as we now don't get a fast signal if one of the tests times out.

It turned out, we don't need any timeout. What we needed to do to fix time timeouts was set the Umi commitment to `confirmed`.